discovery-infra/bootstrap_in_place.py

```diff
@@ -74,8 +75,8 @@ def download_live_image(download_path):
     logging.info("Downloading iso to %s", download_path)
     # TODO: enable fetching the appropriate rhcos image
     utils.run_command(
-        f"curl https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/"
-        f"4.7.0-rc.2/rhcos-4.7.0-rc.2-x86_64-live.x86_64.iso --retry 5 -o {download_path}")
+        f"curl https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.2/rhcos-live.x86_64.iso"
+        f" --retry 5 -o {download_path}")
```

Comment:
Add `--continue-at -` to retry from last position. Increase retry attempts to around ~10-12. Retries happen with 1 second exponential backoff, so 5 retries is just 1 + 2 + 4 + 8 + 16 (31) seconds, not enough time IMO to let a server recover from failure